<launch>
  <rosparam file="$(find ghattas_control)/config/parameters.yaml" command="load"/>
  <node name="heading_control" pkg="ghattas_control" type="heading_control.py" output="screen"/>
  <node name="depth_control" pkg="ghattas_control" type="depth_control.py" output="screen"/>
  <node name="control_maintainer" pkg="ghattas_control" type="control_maintainer.py" output="screen"/>
  <node name="move" pkg="ghattas_control" type="move.py" output="screen"/>
  <!-- <node name="long_lat_control" pkg="ghattas_control_advanced" type="long_lat_control.py" output="screen"/> -->
  <node name="save_vehicle_state" pkg="ghattas_control" type="save_vehicle_state.py" output="screen"/>
</launch>
